Description: fix a bogus Java version check
 These clowns look at just the first digit, thinking that 10 means 1.

--- db5.3-5.3.28+dfsg1.orig/dist/configure.ac
+++ db5.3-5.3.28+dfsg1/dist/configure.ac
@@ -453,11 +453,11 @@ if test "$db_cv_java" = "yes" -o "$db_cv
         	       	sed -e '/ version /!d' -e 's/.*"\(.*\)".*/\1/'` ;;
 	esac
 	AC_MSG_RESULT($JAVA_VERSION)
-	case "$JAVA_VERSION" in
-	1.[[3456789]]* | 1.[[1-9]][[0-9]]* | [[23456789]]* ) ;;
-	* )
-		AC_MSG_ERROR([Java version 1.3 or higher required, got $JAVA_VERSION]) ;;
-	esac
+	#case "$JAVA_VERSION" in
+	#1.[[3456789]]* | 1.[[1-9]][[0-9]]* | [[23456789]]* ) ;;
+	#* )
+	#	AC_MSG_ERROR([Java version 1.3 or higher required, got $JAVA_VERSION]) ;;
+	#esac
 
 	# Because of the code that SWIG generates to cast between pointers and
 	# integers, we need to add the flag "-fno-strict-aliasing" to the gcc
